algotaurus.py

```diff
@@ -468,6 +471,13 @@ def __init__(self, size=15, lines=30):
 GOTO x\t      Continue with line x''')
 
         # Create menu for the GUI
+        languages={_('Hungarian'):'hu', _('English'):'en'}
```

Comment:
Language dictionary: only this variable has to be modified when we want to add more languages